dir.props

```diff
@@ -13,8 +13,8 @@
   <PropertyGroup>
     <MajorVersion>2</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <PatchVersion>1</PatchVersion>
```

Comment:
Actually for those 2 packages I would like to do something similar to corefx and disable the packaging of them instead of bumping them. I would prefer we not ship them with every servicing release.